payments-core/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -96,8 +96,8 @@ constructor(
      */
     private val threeDs1IntentReturnUrlMap = mutableMapOf<String, String>()
 
-    private val authenticatorRegistry: PaymentAuthenticatorRegistry =
-        DefaultPaymentAuthenticatorRegistry.createInstance(
+    private val authenticatorRegistry: PaymentNextActionHandlerRegistry =
```

Comment:
should `authenticatorRegistry` also be renamed?

Reply:
if the answer is yes, please feel free to do this in a follow up!

Reply:
I missed that, just pushed a fix for it